### ISSUE TITLE
ES modules don't work on Windows

### ISSUE THREAD

**Issue by JannesMeyer** [jasmine/jasmine-npm#169]
I tried to use the new ESM support by creating an .mjs file and running it with Jasmine.

However, I get the following error:

```
Error [ERR_UNSUPPORTED_ESM_URL_SCHEME]: Only file and data URLs are supported
by the default ESM loader. On Windows, absolute paths must be valid file:// URLs.
Received protocol 'c:'
```

It seems that the generated file name is not compatible with the Windows version of node.

The documentation says that [absolute paths have to be file: URLs](https://nodejs.org/docs/latest/api/esm.html#esm_import_specifiers).

OS: Windows 10 (Build 18363.1129)
Node: v15.0.1
Jasmine: v3.6.2

**Comment by sgravrock** [comment]
This is fixed in 3.6.3. Thanks for the report.

**Comment by JannesMeyer** [comment]
Wonderful, thanks!
